runtime/Cargo.toml

```diff
@@ -4,7 +4,7 @@ edition = '2018'
 license = 'Apache-2.0'
 repository = 'https://github.com/digicatapult/dscp-node/'
 name = 'dscp-node-runtime'
-version = '3.0.1'
+version = '3.1.0'
```

Comment:
we should probably bump the version in lib.rs as well. Just because we have now redoployed the cluster so the current version is that in staging